src/components/Cards.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+
+<div class="not-prose flex flex-col md:flex-row pt-4 md:space-x-6 lg:space-x-12 pb-16 space-y-6 md:space-y-0">
+<span
+  v-for="card in cards">
+  <Card
```

Comment:
Can you do a lot of this with slots? Card is a very generic name for a component so forcing all content to go through props means it's not flexible at all

Reply:
I have updated to use slot